src/app/customer/Customer.tsx

```diff
@@ -159,9 +170,35 @@ class Customer extends Component<CustomerProps & WithCheckoutCustomerProps, Cust
         });
     };
 
+    private renderCreateAccountForm(): ReactNode {
+        const {
+            customerAccountFields,
+            isCreatingAccount,
+            createAccountError,
+            canCreateAccountOnCheckout,
+            createAccountLink,
+            isEmbedded,
+        } = this.props;
+
+        if (isEmbedded || !canCreateAccountOnCheckout) {
+            window.top.location.assign(createAccountLink);
```

Comment:
Do you want to render a link which redirects the shopper to the sign up page when it is clicked? Or simply exit early and not render anything. At the moment, it seems you're directly doing a redirect in a `render` method.

Reply:
I'm after the latter. If account creation in the checkout is not supported, the user has already clicked a link. This is for keeping the logic centralized in one place, rather than updating "create account" links to behave differently depending on an experiment.

Reply:
Latter as in "not render anything" or you want to actually do a redirect here. If it's "not render anything", just do `return`. If it's "do a redirect here", I recommend you doing it elsewhere because it's good practice for `render` to not result in any side effect as the method can be called multiple times by React.